XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -1133,7 +1168,23 @@ private void toggleConsoleButton_Click(object sender, EventArgs e)
 
                     vncScreen.UseVNC = !vncScreen.UseVNC;
                     vncScreen.UserWantsToSwitchProtocol = true;

```

Comment:
Yes, I think it is because user already made choice of using which kind of connection, if the decision has already been made, change of the status of RDP shall not affect the connection status. Am I right?
Same logic is also here:  https://github.com/xenserver/xenadmin/blob/f806228923acd7a20656e759c661040a8f0c763a/XenAdmin/ConsoleView/VNCTabView.cs#L1082